modules/user_accounts/php/edit_user.class.inc

```diff
@@ -293,9 +290,16 @@ class Edit_User extends \NDB_Form
             // START EXAMINER UPDATE
             // If examiner not in table add him,
             // otherwise update the radiologist field and get the current sites
-            $values = \Utility::nullifyEmpty($values, 'examiner_radiologist');
-            if (empty($examinerID)) {
-
+            if (!empty($ex_radiologist)
+                && !empty($ex_pending)
+                && !empty($ex_curr_sites)
+                && empty($examinerID)
+            ) {
+                $ex_radiologist = $ex_radiologist === 'Y' ? 1 : 0;
+                $values         = \Utility::nullifyEmpty(
```

Comment:
This function call is useless here, we assigned `$values[examiner_radiologist]` to `$ex_radiologist` earlier and then checked for `!empty($ex_radiologist)`. The call should be made outside of this `if` statement.
I also don't see the point of making this call at all, the value is not used again and it's unset after this code.